### ISSUE TITLE
Not connecting

### ISSUE THREAD

**Issue by landshark22** [RocketChat/Rocket.Chat#5159]
Your Rocket.Chat version: 0.47

I am trying to get Rocket.Chat working on my server but can't seem to get it to load in the browser or app. 

I have a dedicated server running CentOS 6 (WHM/Cpanel) with several sites hosted. I would like to use RC with one of the domains/sites. 

I've installed Mongo/Rocket.Chat using the instructions here. https://rocket.chat/docs/installation/manual-installation/centos/

When I navigate to the URL the page times out and same with the Windows App, I get the error "timed out trying to connect" 

I unblocked port 3000 in CSF, at least I think I did. 

When I run the main.js (using PuTTY) i get the following

[root@host /opt/Rocket.Chat]# node main.js
Updating process.env.MAIL_URL
ufs: permissions are not defined for store "fileSystem"
ufs: store created at
ufs: permissions are not defined for store "fileSystem"
ufs: store created at
{"line":"160","file":"rocketchat_migrations.js","message":"Migrations: Not migrating, already at version 68","time":{"$date":1481265563647},"level":"info"}
Updating process.env.MAIL_URL
Using GridFS for custom emoji storage
[Fri Dec 09 2016 01:39:25 GMT-0500 (EST)] WARNING A script has tried registering a HTTP route while the HTTP server is disabled with --disabled-httpd.
Loaded hubot-help/src/help.coffee
Loaded hello.coffee
Loaded zen.coffee
ufs: permissions are not defined for store "rocketchat_uploads"
Using GridFS for Avatar storage
[Fri Dec 09 2016 01:39:27 GMT-0500 (EST)] ERROR Error: listen EADDRINUSE 0.0.0.0:3000
  at Object.exports._errnoException (util.js:907:11)
  at exports._exceptionWithHostPort (util.js:930:20)
  at Server._listen2 (net.js:1250:14)
  at listen (net.js:1286:10)
  at net.js:1395:9
  at nextTickCallbackWith3Args (node.js:453:9)
  at process._tickDomainCallback (node.js:400:17)

➔ System ➔ startup
➔ +--------------------------------------+
➔ |            SERVER RUNNING            |
➔ +--------------------------------------+
➔ |                                      |
➔ |       Version: 0.47.0                |
➔ |  Process Port: 3000                  |
➔ |      Site URL: http://mysiteURL:3000/  |
➔ |         OpLog: Disabled              |
➔ |                                      |
➔ +--------------------------------------+

Any Ideas what I may be doing wrong? Tests I should run?

**Comment by landshark22** [comment]
Is there anyone that can help or point me to in a direction where I can begin to troubleshoot the problem? My searching has come up empty but I very well be searching the wrong terms. 

Would I be better off having my server upgraded to CentOS 7 and then go from there?

**Comment by landshark22** [comment]
Ok, it is now working now. I killed the process with:

`ps -ax | grep node`

and restarted and it seems to be working. 

I am having another issue but my server is being upgraded to CentoOS 7 so I will create another post after that is completed and if the problem still persists.
